src/applications/personalization/dashboard/components/cerner-widgets.js

```diff
@@ -0,0 +1,128 @@
+import React, { Fragment } from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+import environment from 'platform/utilities/environment';
+import { mhvUrl } from 'platform/site-wide/mhv/utilities';
+
+// Returns an AlertBox to present the user with info about working with the
+// Cerner facility they are enrolled at. Props allow you to edit a small amount
+// of the content that is rendered in the AlertBox.
+const CernerAlertBox = ({
+  ctaButtonText,
+  ctaButtonUrl,
+  ctaText,
+  facilityNames,
+}) => {
+  // Helper component that takes an array of facility names and a separator string and returns some JSX to style the list of facility names.
+  const FacilityList = ({ facilities, separator }) => {
+    const newArray = [];
+    // first make an array that alternates between a facility name and the
+    // separator word wrapped in a <span> for styling purposes
+    facilities.forEach(el => {
+      newArray.push(el);
+      newArray.push(
+        <span className="vads-u-font-weight--normal vads-u-font-size--base">
+          {separator}
+        </span>,
+      );
+    });
+    // we don't need the last separator in the array
+    newArray.pop();
+    // Then map over the array we just made, converting it to JSX
+    return (
+      <>
+        {newArray.map((el, i) => {
+          return <Fragment key={i}>{el}</Fragment>;
+        })}
+      </>
+    );
+  };
+
+  return (
+    <AlertBox
+      status="warning"
+      headline="Your VA health care team may be using our new My VA Health portal"
+    >
+      <h3>Our records show you’re registered at:</h3>
+      <h4>
+        <FacilityList facilities={facilityNames} separator=" and " />
+        <span className="vads-u-font-weight--normal vads-u-font-size--base">
+          {' '}
+          (Using My VA Health)
+        </span>
+      </h4>
+      <p>
+        Please choose a health management portal below, depending on the
+        facility for your appointment. You may need to disable your browser’s
+        pop-up blocker to open the portal. If you’re prompted to sign in again,
+        use the same account you used to sign in on VA.gov.
+      </p>
+      <h3>{ctaText}</h3>
+      <h4>
+        <FacilityList facilities={facilityNames} separator=" or " />
+      </h4>
+      <a
+        href={
+          environment.isProduction()
+            ? 'https://patientportal.myhealth.va.gov/'
```

Comment:
Can we use `getCernerURL from `src/platform/utilities/cerner/index.js` here?

Reply:
Looks like it! thanks! I was just copying the code from the existing widget